### ISSUE TITLE
Media region for Screenreader users broken in 2.4 RC5 

### ISSUE THREAD

**Issue by WestphalDenn** [bigbluebutton/bigbluebutton#13807]
As a blind user, I compared BBB 2.3 and the RC 5 of BBB 2.4. I really like the numerous improvements. However the media region seems missing and/or misplaced. 

What this means is that there is no easy way to jump to the top of a presentation that is shared in BBB. 

## Steps to reproduce 

* Using a browser of choice and the NVDA Screenreader. 
* Join a BBBMeeting
* When put into the textfield for writing into the public chat, press the insert-key and the spacebar together. You will hear a sound, confirming that you have changed from form mode to browsing mode. 
* Press ctrl+Pos1 to jump to the upper left of the page. 
* Press the letter "D" to navigate by regions. 

## Findings

You will notice that in 2.3 there is a "media region" which places you at the top of the shared presentation or document. 

In 2.4 there is a region called "Document" which will put the focus to the button that minimizes the userlist and the public chat. 

Thanks for improving this.

**Comment by antobinary** [comment]
Hi @WestphalDenn @KDSBrowne 's fix should ship on BBB 2.4 with the upcoming release
